aten/src/ATen/native/native_functions.yaml

```diff
@@ -13488,220 +13488,220 @@
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: expand_copy(Tensor self, SymInt[] size, *, bool implicit=False) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: expand_copy_symint
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: permute_copy(Tensor self, int[] dims) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: permute_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: _reshape_alias_copy(Tensor self, SymInt[] size, SymInt[] stride) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: _reshape_alias_copy_symint
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: select_copy.int(Tensor self, int dim, SymInt index) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: select_copy_symint
     SparseCsrCPU, SparseCsrCUDA: select_copy_sparse_csr
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: detach_copy(Tensor self) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: detach_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: slice_copy.Tensor(Tensor self, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: slice_copy_Tensor_symint
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: split_copy.Tensor(Tensor self, SymInt split_size, int dim=0) -> Tensor[]
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: split_copy_Tensor_symint
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: split_with_sizes_copy(Tensor self, SymInt[] split_sizes, int dim=0) -> Tensor[]
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: split_with_sizes_copy_symint
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: squeeze_copy(Tensor self) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: squeeze_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: squeeze_copy.dim(Tensor self, int dim) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: squeeze_copy_dim
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: squeeze_copy.dims(Tensor self, int[] dim) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: squeeze_copy_dims
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: t_copy(Tensor self) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: t_copy
-  tags: view_copy
+  tags: [canonical, view_copy]
 
 - func: transpose_copy.int(Tensor self, int dim0, int dim1) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: transpose_copy_int
-  tags: view_copy
+  tags: [canonical, view_copy]
```

Comment:
For example, both aten.t and aten.transpose should have been decomposed to aten.permute. 

So if we were to mark the *_copy op, we should only do it fro aten.permute_copy()